### PR TITLE
Logging backend

### PR DESCRIPTION
**Pull request by UwUAroze**
mriaoww~ ;3

### CODE REVIEW

**Comment by UwUAroze** [comment]
also completes #22

File: src/events/handlers/postprocessor/CountingPostProcessor.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    static async handle(
+        deletedMessage: Message,
+        channel: TextChannel|undefined,
+        author: User,
+        loggedMessage: Prisma.LoggedMessageGetPayload<{ include: { attachments: true } }>,
+        countData: GuildCounting,
+    ) {
+        if (loggedMessage.authorId != countData.lastCounterId) return
+        if (evaluate(loggedMessage.textContent) !== countData.count) return
+        await channel?.send(CountingHandler.parseCountingMessage(countData.caughtDeleteMessage, author, countData.count))
```
Comment (lilysoftpaws) [inline src/events/handlers/postprocessor/CountingPostProcessor.ts:22]: && exists

**Review by lilysoftpaws — APPROVED** [review]
lgtm ⋆˚.⋆ ˚⋆｡ maybe message logging could be revamped at a later date but it works for now ^_^

**Comment by UwUAroze** [comment]
took you long enough..
thanks mommy~ ;3

**Comment by Dawsson** [comment]
Did you just call ur partner mommy in a github issue

**Comment by UwUAroze** [comment]
m..meow~

**Pull request merged**